PWGLF/Tasks/lambda1520analysis.cxx

```diff
@@ -371,4 +371,4 @@ struct lambda1520analysis {
 WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
 {
   return WorkflowSpec{adaptAnalysisTask<lambda1520analysis>(cfgc, TaskName{"lf-lambda1520analysis"})};
-}
+}
```

Comment:
This line has to be reverted. Make a new line break at the end of the file.

Reply:
Done.